tests/snuba/models/test_event.py

```diff
@@ -78,26 +77,3 @@ def test_bind_nodes(self):
         event.bind_node_data()
         assert event.data._node_data is not None
         assert event.data["user"]["id"] == u"user1"
-
-    def test_event_with_no_body(self):
```

Comment:
Since we don't have sampling as a feature anymore, I don't think this is a valid use case, and much of our product already depends on the events being present in nodestore and would otherwise be breaking if the event bodies were missing.